src/test/java/com/example/country/CountryControllerTest.java

```diff
@@ -0,0 +1,155 @@
+package com.example.country;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.patch;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(CountryController.class)
+class CountryControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private CountryService countryService;
+
+    @Test
+    public void 全ての国を取得する() throws Exception {
```

Comment:
テストケースですが、
`できること`、`できないこと`といったようにまとめてもらうといいですね。
```suggestion
    public void 全ての国を取得できること() throws Exception {
```

Reply:
dbcec90d378dd3863609996a375af49bda1e14cc
ご確認ありがとうございます。
テストケースを修正いたしました。